landing_page/app/javascript/components/Home.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import CSS from 'csstype';
+
+type HomeProps = {
+    title:string
+    content:string
+};
+
+const h1Styles: CSS.Properties = {
```

Comment:
I would rather see css styles defined in .css  files rather than in the react component itself. It is useful to implement this separation of concerns - structure and logic of the component within the .tsx file and styling in .scss or .css files. In this way different styles can be easily implemented by swapping in .sccs files and the component is not tied to one particular style